.rubocop.yml

```diff
@@ -65,6 +67,12 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Exclude:
     - 'spec/**/*.rb'
+    - 'lib/haml_lint/spec/**/*.rb'
+
+Metrics/ModuleLength:
```

Comment:
Unfortunately a couple lines over because of a couple whitespace changes to make line length work.